### PR TITLE
troubleshooting(webcontainer): how to fix infinite reload in chrome

### PR DESCRIPTION
**Pull request by Nemikolh**
# PR Description

This PR provides some guidance on how to resolve the "Infinite reload page" issue.

This is an issue where when you open chrome's devtools on the preview page, doing a refresh will force the service worker to be re-installed. In doing so, it will:

 1. Take control of the page, refreshing it
 2. The page being refreshed and "update on reload" being checked, reinstall the service worker

Thus the loop. :disguised_face: 

### Summary of my changes and explanation of my decisions

We add a section to troubleshooting explaining the step to follow to prevent that error: disable the `update on reload` option.

### Self-check

Please check all that apply:

- [x] My code follows the style guidelines of this project
- [x] I have reviewed my code or content update and edited it to the best of my abilities
- [ ] I have commented my code, particularly in hard-to-understand areas; my comments are concise

### CODE REVIEW

File: docs/platform/webcontainers/troubleshooting-webcontainers.md [hunk 1]
```diff
@@ -27,3 +27,19 @@ With the release of native npm support, the `npm` command in a terminal no longe
  3. Run `npm install --force` to bypass the conflicts.
 
 Your project should load without your action, at least if dependencies are automatically installed. If an error is detected, StackBlitz will automatically re-run the installation step with `npm install --legacy-peer-deps`. You can find more information on how to configure your project [here](https://developer.stackblitz.com/platform/webcontainers/project-config).
+
+## Chrome: Infinite reload when dev tools are opened
```
Comment (d3lm) [inline docs/platform/webcontainers/troubleshooting-webcontainers.md:31]:
```suggestion
## Chrome: Infinite reload when DevTools are opened
```

File: docs/platform/webcontainers/troubleshooting-webcontainers.md [hunk 2]
```diff
@@ -27,3 +27,19 @@ With the release of native npm support, the `npm` command in a terminal no longe
...
+
+If you have your preview in a separate tab and you open Chrome dev tools, you can observe an infinite reload loop after a refresh. This is usually caused by having the `Update on reload` checkbox checked in `Application > Service Workers`.
```
Comment (d3lm) [inline docs/platform/webcontainers/troubleshooting-webcontainers.md:33]: Maybe we can slightly rephrase? Just an idea:
```suggestion
You might find yourself stuck in an infinite reload loop upon refreshing your webpage. This is usually caused by having the `Update on reload` setting checked under `Application > Service Workers` if you've launched your preview in a separate tab while Chrome DevTools are open.
```

File: docs/platform/webcontainers/troubleshooting-webcontainers.md [hunk 3]
```diff
@@ -27,3 +27,19 @@ With the release of native npm support, the `npm` command in a terminal no longe
...
+
+<img alt="Chrome DevTools with the Application tab selected showing the Service Workers section with the Update on reload checkbox checked" src="./assets/chrome-update-on-reload.png" width="800" />
```
Comment (d3lm) [inline docs/platform/webcontainers/troubleshooting-webcontainers.md:35]: Nice!

File: docs/platform/webcontainers/troubleshooting-webcontainers.md [hunk 4]
```diff
@@ -27,3 +27,19 @@ With the release of native npm support, the `npm` command in a terminal no longe
...
+
+To fix this issue, simply disable that option.
+
+:::info
+
+At this time, it is not possible to install your own service workers with webcontainer as our own service worker is used as a core component of the networking stack.
```
Comment (d3lm) [inline docs/platform/webcontainers/troubleshooting-webcontainers.md:41]:
```suggestion
At this time, it is not possible to install your own Service Worker with WebContainer as our own Service Worker is used as a core component of the networking stack.
```

File: docs/platform/webcontainers/troubleshooting-webcontainers.md [hunk 5]
```diff
@@ -27,3 +27,19 @@ With the release of native npm support, the `npm` command in a terminal no longe
...
+
+Having this option enabled mean it update webcontainer's service worker on every page refresh.
```
Comment (d3lm) [inline docs/platform/webcontainers/troubleshooting-webcontainers.md:43]:
```suggestion
Having this option enabled means it updates WebContainer's Service Worker on every page refresh.
```

Comment (d3lm) [inline docs/platform/webcontainers/troubleshooting-webcontainers.md:43]: Maybe we can mention why that is an issue. Also fine if we don't.

**Review by d3lm — COMMENTED** [review]
Nice! Thanks 🙏

**Review by d3lm — APPROVED** [review]

**Comment by sylwiavargas** [comment]
🥰

**Pull request merged**